server.js

```diff
@@ -1,5 +1,69 @@
 // import your node modules
-
+const express = require('express')
+const bodyParser = require('body-parser')
 const db = require('./data/db.js');
 
 // add your server code starting here
+const server = express();
+server.use(bodyParser.json())
+server.use(express.json());
+
+
+server.get('/api/posts',(req,res) => {
+    db.find().then((posts) => {
+        res.status(200).json(posts)
+    })
+})
+
+server.get('/api/posts/:id',(req,res) => {
+    db.findById(req.params.id).then( (post)=>
+        {res.status(200).json(post)}
```

Comment:
We need two error handlers here. 404 if post with specified id wasn't found, and 500 if `db.findById()` Promise was rejected.